config-linux.md

```diff
@@ -270,13 +270,16 @@ For more information, see the kernel cgroups documentation about [memory][cgroup
 **`memory`** (object, OPTIONAL) represents the cgroup subsystem `memory` and it's used to set limits on the container's memory usage.
 For more information, see the kernel cgroups documentation about [memory][cgroup-v1-memory].
 
-The following parameters can be specified to set up the controller:
+Values for memory specify the limit in bytes, or `-1` for unlimited memory.
+
+* **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage
+* **`reservation`** *(int64, OPTIONAL)* - sets soft limit of memory usage
+* **`swap`** *(int64, OPTIONAL)* - sets limit of memory+Swap usage
+* **`kernel`** *(int64, OPTIONAL)* - sets hard limit for kernel memory
+* **`kernelTCP`** *(int64, OPTIONAL)* - sets hard limit for kernel TCP buffer memory
+
+For `swappiness` the values are from `0` (almost never swap) to `100` (very likely to be swapped).
```

Comment:
Is 0 really “*almost* never”?  From [the cgroup docs][1]:

> Please note that unlike during the global reclaim, limit reclaim enforces that 0 swappiness really prevents from any swapping even if there is a swap storage available.

which seems stronger than the [the sysctl docs][2]:

> A value of 0 instructs the kernel not to initiate swap until the amount of free and file-backed pages is less than the high water mark in a zone.

And neither one of those mentions a max at 100.  I don't doubt that 100 is the current max, but I'd rather punt to kernel docs for the `swappiness` semantics instead of inlining them with this line.

[1]: https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
[2]: https://www.kernel.org/doc/Documentation/sysctl/vm.txt